hackathon_site/dashboard/frontend/src/pages/Teams/Teams.module.scss

```diff
@@ -0,0 +1,29 @@
+@import "./../../assets/abstracts/mixins";
+@import "./../../assets/abstracts/variables";
+
+.dashboard {
```

Comment:
I don't think you use .dashboard style in the page so you can remove it

for .dashboardGrid I think you can remove all the styling there too bc it makes more sense that all the cards are evenly spaced, without margin on the rows

For .dashboardGrid Item I think the only thing needed here is the padding (since the minHeight is on the team card now) but since its just one style, you can put it in the tsx file and remove all the styles from here.

Reply:
I simplified it

Reply:
why are you making each card have a max-width of 1400px?

Reply:
also if you want to use this class, could you name it to something related to the team card because "dashboard" is misleading since it's not the dashboard page